BigBlueButton-RLP/Constants/Script.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  Script.swift
+//  BigBlueButton-RLP
+//
+//  Created by Milan Bojic on 20.12.21..
+//
+
+import Foundation
+
+struct Script {
```

Comment:
Could/should this be an `enum`? https://stackoverflow.com/questions/38585344/swift-constants-struct-or-enum

Reply:
Yes, good point, dont know why I didnt make it enum :)